docs/codeql/support/notes-ql-users.rst

```diff
@@ -1,91 +0,0 @@
-Notes for legacy QL CLI users
```

Comment:
This was useful for a few straggling customers, but I think they've all moved off the legacy CLI now.